e2e-tests/functions

```diff
@@ -76,6 +88,17 @@ get_cr() {
 				mv "${TEMP_DIR}/start-from-backup.cr.yaml" "${TEMP_DIR}/cr.yaml"
 			fi
 			;;
+		scheduled-backup)
+			yq eval '.spec.backups.pgbackrest.configuration = [{"secret":{"name":"'${test_name}'-pgbackrest-secrets"}}]' "${TEMP_DIR}/cr.yaml" \
```

Comment:
`yq` supports writing multiple commands in a single command, something like:
```
    yq eval '
        .metadata.name = "chaos-cluster-pod-kill" |
        del(.spec.selector.pods.test-namespace) |
        .spec.selector.pods.'$namespace'[0] = "'$pod'"' $conf_dir/chaos-pod-kill.yml \
        | kubectl apply -f -
```
might be a bit shorter and faster then piping